internal/policy/input_test.go

```diff
@@ -1,90 +0,0 @@
-// Copyright 2022 Red Hat, Inc.
```

Comment:
We're missing these tests in the refactor

Reply:
Missing as in unintentionally dropped.. or something to add later?